connector_node/risingwave-source-test/src/test/java/com/risingwave/connector/MySQLSourceTest.java

```diff
@@ -0,0 +1,192 @@
+                            connection, testClient.sqlStmts.getProperty("mysql.bin_log"));
+            assertThat(resultSet.getString("Value")).isEqualTo("ON").as("MySQL: bin_log ON");
+            connection.close();
+        } catch (SQLException e) {
+            fail("SQL exception: ", e);
+        }
+    }
+
+    @AfterClass
+    public static void cleanup() {
+        connectorServer.shutdown();
+        mysql.stop();
+    }
+
+    @Test
+    public void testLines() throws InterruptedException, SQLException {
+        Lock lock = new ReentrantLock();
+        Condition done = lock.newCondition();
+        Connection connection = SourceTestClient.connect(mysql);
+        String query = testClient.sqlStmts.getProperty("tpch.create.orders");
+        SourceTestClient.performQuery(connection, query);
+        query =
+                "LOAD DATA INFILE '/home/orders.tbl' "
+                        + "INTO TABLE orders "
+                        + "CHARACTER SET UTF8 "
+                        + "FIELDS TERMINATED BY '|' LINES TERMINATED BY '\n';";
+        SourceTestClient.performQuery(connection, query);
+        Iterator<GetEventStreamResponse> eventStream =
+                testClient.getEventStreamStart(mysql, SourceType.MYSQL, "test", "orders");
+        AtomicInteger count = new AtomicInteger();
+        Thread t1 =
+                new Thread(
+                        () -> {
+                            while (eventStream.hasNext()) {
+                                List<CdcMessage> messages = eventStream.next().getEventsList();
+                                for (CdcMessage ignored : messages) {
+                                    count.getAndIncrement();
+                                }
+                                if (count.get() == 10000) {
+                                    lock.lock();
+                                    try {
+                                        done.signal();
+                                    } finally {
+                                        lock.unlock();
+                                    }
+                                }
+                            }
+                        });
+        t1.start();
+        lock.lock();
+        try {
+            done.await();
+        } finally {
+            lock.unlock();
+        }
+        logger.info("count: {}", count.get());
+        assertThat(count.get()).isEqualTo(10000);
+        connection.close();
+    }
+
+    @Ignore
+    @Test
+    public void getTestJson() throws InterruptedException, SQLException {
+        Connection connection = SourceTestClient.connect(mysql);
+        String query =
+                "CREATE TABLE IF NOT EXISTS orders ("
+                        + "O_KEY BIGINT NOT NULL, "
+                        + "O_BOOL BOOLEAN, "
+                        + "O_TINY TINYINT, "
+                        + "O_INT INT, "
+                        + "O_REAL REAL, "
+                        + "O_DOUBLE DOUBLE, "
```

Comment:
please cover overflow cases

Reply:
as for the overflow cases...
when inserting data into mysql, if there is overflow, mysql will throw an error:
> Data truncation: Out of range value for column 'O_INT' at row 1
so it's possibly unlikely that debezium will generate overflowing values...?

I can also try to modify the json generated by debezium and see if the parser catches the error

Reply:
debezium is unlikely to deliver the corrupted data but we still may face it when the data comes from kafka.
Just mock some test cases and see if rw catches the error.